### PR TITLE
Add Terms&Privacy+replace avatar+name on settings

### PR DESCRIPTION
**Pull request by BlockJayn**
Add/settings-screen
- Added Terms and Conditions to bottom of screen
- Replaced Avatar with obi-logo
- Replaced Name with "Obi ..."
- Hide Subtext of name
- Hide Account settings (due to current decision)

Simulator Screen Shot - iPhone 13 Pro Max - 2022-09-07 at 20 39 35

### CODE REVIEW

File: obi-wallet/apps/mobile/src/app/screens/account/index.tsx [hunk 1]
```diff
@@ -91,26 +93,80 @@ export function Account() {
...
+              justifyContent: "flex-start",
             }}
           >
-            $38,166.92
-          </Text>
+            <View
+              style={{
+                flexDirection: "row",
+                alignItems: "center",
+                //backgroundColor: "red",
```
Comment (inyono) [inline obi-wallet/apps/mobile/src/app/screens/account/index.tsx:107]: Unused code I suppose.

File: obi-wallet/apps/mobile/src/app/screens/account/index.tsx [hunk 2]
```diff
@@ -91,26 +93,80 @@ export function Account() {
...
+                />
+              </TouchableOpacity>
+
+              <View style={{ flexDirection: "column" }}>
+                <Text
+                  style={{
+                    color: "#F6F5FF",
+                    fontSize: 18,
+                    fontWeight: "700",
+                    fontFamily: "Inter",
```
Comment (inyono) [inline obi-wallet/apps/mobile/src/app/screens/account/index.tsx:132]: Inter is already the default font family used by our `Text` component, so not necessary.

File: obi-wallet/apps/mobile/src/app/screens/settings/index.tsx [hunk 3]
```diff
@@ -146,8 +177,49 @@ export function SettingsScreen() {
...
+              fontSize: 10,
+            }}
+          >
+            Privacy Policy
+          </Text>
+        </View>
+      </View>
+
+      {/*<View style={[styles.flex1]}></View>*/}
+      {/*<View style={[styles.flex1]}></View>*/}
```
Comment (inyono) [inline obi-wallet/apps/mobile/src/app/screens/settings/index.tsx:222]: unused code?

**Review by inyono — CHANGES_REQUESTED** [review]

**Comment by BlockJayn** [comment]
Thx for the feedback @inyono, requested changes are done

**Review by inyono — APPROVED** [review]
Thanks :)

**Pull request merged**